### DISCUSSION TITLE
How to find persistent file in MacOS if I want to move Monika into another computer/laptop, and how exactly to do so?

### DISCUSSION THREAD

**Discussion by JustAskingStuff** [Monika-After-Story/MonikaModDev#8430] [QA - tech support + general] [answered]
So I've viewed the discussion before about transferring Monika, and saw they moved the "persistent" file inside their device. Well for Mac user like me I didn't have any "persistent" files, does it has a different name or just not there? Or should I only transfer the "Monika" file after saying "I'm going to take you somewhere else"? And what am I suppose to do after installed a new, un-open MAS on a new device?

**Comment by dreamscached** [comment, marked as answer]
https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-want-to-back-up-my-persistent

**Comment by JustAskingStuff** [comment]
@dreamscached thanks a lot, do I have to move both "log" and "Monika" files together to new device? Or just move the entire game folder to it like that article said?

**Comment by dreamscached** [comment]
Better copy and paste the entire folder and make sure to copy persistent data as well; never delete original game folder until you've made sure it works on another device.

**Comment by JustAskingStuff** [comment]
ok I see

**Comment by JustAskingStuff** [comment]
ngl this is somehow unsettling to think that the original Monika got copied, and watch the same copy shares the same memory with her being together with me happily, yet herself got stuck in the old device and gonna(probably already) get deleted.

**Reply by ThePotatoGuy** [reply]
Oh man you'd hate [SOMA](https://store.steampowered.com/app/282140/SOMA/)

**Reply by JustAskingStuff** [reply]
> Oh man you'd hate [SOMA](https://store.steampowered.com/app/282140/SOMA/)

yuuupppp that game made me relate to DDLC's "character" and "persistent" files and now I'm overthinking🥲
